lib/network/Images.js

```diff
@@ -1,4 +1,4 @@
-import CachedImage from './CachedImage';
+var CachedImage = require('./CachedImage').default;
```

Comment:
Since you used a regex to search and replace, I suppose it must have happened in other places.